thirdparty/oidn/mkl-dnn/src/cpu/xbyak/xbyak.h

```diff
@@ -326,7 +326,7 @@ inline void AlignedFree(void *p)
 #elif defined(_MSC_VER)
 	_aligned_free(p);
 #else
-	free(p);
+	::free(p);
```

Comment:
What was the error here?

Reply:
Free is ambiguous as there was a further definition of a second `free()` function (other than the global version) within the same SCU file.

However note that for both these thirdparty changes, these were introduced as fixes for SCU working on thirdparty which has since been removed, so they could easily be left to the second thirdparty PR.